service/controller/resource/tccpoutputs/create.go

```diff
@@ -72,10 +72,18 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 	if r.route53Enabled {
 		{
 			v, err := cloudFormation.GetOutputValue(outputs, HostedZoneID)
-			if err != nil {
-				return microerror.Mask(err)
+			// migration code to dont throw error  when the old C dont yet have the new output value
+			// TODO
```

Comment:
TODO issue please.

Reply:
Heh I forgot to push the updated comment :trollface: